Desktop version fails to build ('openssl_fips' is not defined)

the error occurs while trying to build `abandonware/bluetooth-hci-socket@0.5.3-8` from source:

`npm ERR! gyp: name 'openssl_fips' is not defined while evaluating condition 'openssl_fips != ""' in binding.gyp while trying to load binding.gyp`

relevant node-gyp issue thread:
https://github.com/nodejs/node-gyp/issues/2673

dependency tree:
```shell
npm ls @abandonware/bluetooth-hci-socket@0.5.3-8
nault@1.18.0
`-- @ledgerhq/hw-transport-node-ble@6.27.1
  `-- @abandonware/noble@1.9.2-15
    `-- @abandonware/bluetooth-hci-socket@0.5.3-8
```

--- a/.github/workflows/electron_hash.yml
+++ b/.github/workflows/electron_hash.yml
@@ -33,9 +33,11 @@ jobs:
 
   release:
     runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
 
     # Run different jobs for each OS
     strategy:
+      fail-fast: false
       matrix:
         include:
           - os: ubuntu-20.04
@@ -43,16 +45,19 @@ jobs:
             asset_name: checksums
             app_ext: AppImage
             app_os: linux
+            experimental: false
           - os: macos-11
             artifact_name: checksums
             asset_name: checksums
             app_ext: dmg
             app_os: mac
+            experimental: false
           - os: windows-2022
             artifact_name: checksums
             asset_name: checksums
             app_ext: exe
             app_os: windows
+            experimental: true
 
     needs: pre_build # job dependency and shared variables
     steps:

--- a/.github/workflows/electron_hash.yml
+++ b/.github/workflows/electron_hash.yml
@@ -81,7 +81,7 @@ jobs:
 
       - name: Build Electron app
         run: |
-          npm run desktop:local
+          npm run desktop:local --openssl_fips=''
 
       - name: Create Hashes
         if: startsWith(github.ref, 'refs/tags/')

--- a/DEVELOPMENT.md
+++ b/DEVELOPMENT.md
@@ -57,7 +57,7 @@ Then you can just go to the debug screen and choose "Electron: Main", "Electron:
 
 Build the desktop wallet for your local OS (Will be in `desktop-app\build`):
 ```bash
-npm run desktop:local
+npm run desktop:local --openssl_fips=''
 ```
 
 Can also run a complete build for your local OS. The "wallet:build-desktop" command is run automatically. (Will be in `desktop-app\build`):

--- a/DEVELOPMENT.md
+++ b/DEVELOPMENT.md
@@ -22,15 +22,12 @@ npm install
 npm run wallet:dev
 ```
 
-If you want to debug in VS code, first install [debugger for chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome)
-Then you can just go to the debug screen and choose "Launch Chrome http"
+You can use JavaScript debugger bundled with VS Code to debug. Refer [here](https://github.com/microsoft/vscode-js-debug) for additional information.
 
 #### Run the wallet in dev mode as https (for example if using the Ledger device)
 ```bash
 npm run wallet:dev-ssl
 ```
-
-To debug in VS code: Go to debug screen and choose "Launch Chrome https"
 
 ## Build Wallet (For Production)
 Build a production version of the wallet for web:
@@ -52,8 +49,7 @@ Run the desktop wallet in dev mode:
 npm run desktop:dev
 ```
 
-If you want to debug in VS code, first install [debugger for chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome)
-Then you can just go to the debug screen and choose "Electron: Main", "Electron: Renderer", or "Electron: All" for both Main and Renderer threads.
+You can use JavaScript debugger bundled with VS Code to debug. Refer [here](https://github.com/microsoft/vscode-js-debug) for additional information.
 
 Build the desktop wallet for your local OS (Will be in `desktop-app\build`):
 ```bash

--- a/README.md
+++ b/README.md
@@ -14,7 +14,7 @@ Seamless integration with any nano compatible RPC backend/websocket and the aim 
 ___
 
 ## How To Use
-Nault comes in different flavors to suit your need.
+Nault comes in different flavors to suit your needs.
 #### Desktop App
 Available for Windows/Mac/Linux - just head over to the [latest release](https://github.com/Nault/Nault/releases/latest) and download the version for your OS. Arch Linux users may [install it from the (unofficial) AUR](https://aur.archlinux.org/packages/nault-bin/).
 

--- a/README.md
+++ b/README.md
@@ -4,7 +4,7 @@
 [![GitHub All Releases](https://img.shields.io/github/downloads/nault/nault/total)](https://github.com/Nault/Nault/releases/latest)
 [![Discord](https://img.shields.io/badge/discord-join%20chat-orange.svg?logo=discord&color=7289DA)](https://discord.nanocenter.org)
 
-Nault is a community driven fork of the popular nano wallet [NanoVault](https://github.com/cronoh/nanovault) 💙
+Nault is a community-driven fork of the popular nano wallet [NanoVault](https://github.com/cronoh/nanovault) 💙
 
 It's a fully client-side signing wallet for sending and receiving [nano](https://github.com/nanocurrency/nano-node/) either directly in your browser at [nault.cc](https://nault.cc) or with the [desktop app](https://github.com/Nault/Nault/releases/latest).
 

--- a/README.md
+++ b/README.md
@@ -16,7 +16,7 @@ ___
 ## How To Use
 Nault comes in different flavors to suit your needs.
 #### Desktop App
-Available for Windows/Mac/Linux - just head over to the [latest release](https://github.com/Nault/Nault/releases/latest) and download the version for your OS. Arch Linux users may [install it from the (unofficial) AUR](https://aur.archlinux.org/packages/nault-bin/).
+Available for Windows/Mac/Linux – just head over to the [latest release](https://github.com/Nault/Nault/releases/latest) and download the version for your OS. Arch Linux users may [install it from the (unofficial) AUR](https://aur.archlinux.org/packages/nault-bin/).
 
 If you want to verify the binary checksum there are plenty of apps to do this. One way is using a powershell or bash terminal:
 

